ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -161,6 +208,30 @@ pub(crate) enum Operand {
     Unimplemented(#[deku(until = "|v: &u8| *v == 0", map = "deserialise_string")] String),
 }
 
+impl Operand {
+    /// If the operand is a `LocalVariable`, then return the instruction that defines it, or else
+    /// None.
+    ///
+    /// OPT: This is expensive.
+    pub(crate) fn to_instr<'a>(&self, aotmod: &'a Module) -> Option<&'a Instruction> {
+        match self {
+            Self::LocalVariable(lvo) => {
+                Some(&aotmod.funcs[lvo.func_idx].blocks[lvo.bb_idx].instrs[lvo.inst_idx])
+            }
+            _ => None,
+        }
+    }
+
+    /// Return the `InstructionID` of a local variable operand. Must not be called on other
+    /// operands.
+    pub(crate) fn to_instr_id(&self) -> InstructionID {
+        match self {
+            Self::LocalVariable(lvo) => InstructionID::new(lvo.func_idx, lvo.bb_idx, lvo.inst_idx),
+            _ => unreachable!(),
```

Comment:
This arm is definitely reachable, but to do so would be UB, so we want to abort execution. This should really then be a `panic` IMHO.

Reply:
Fixed in f7a664064c4be6065593565929309acb3bf126cf